src/main/java/mods/eln/transparentnode/ThermalHeatExchanger.kt

```diff
@@ -183,6 +184,15 @@ class ThermalHeatExchangerElement(
             thermalPairs.add(ThermalPairing(ic2hotcoolant, ic2coolant, 640.0 / 7.0, 9, 1.0, false))
         }
 
+        if (ic2hotwater != null) {
```

Comment:
This isn't supposed to be here either but it's a nice little feature.
Adds the ability to convert IC2 hotspring water into normal water and gain a little heat in the process.